docs/resources/oxide_vpc_firewall_rules.md

```diff
@@ -77,12 +77,6 @@ Required:
 - `status` (String) Whether this rule is in effect. Possible values are: enabled or disabled.
 - `targets` (Set) Sets of instances that the rule applies to. (see [below for nested schema](#nestedatt--targets))
 
-Read-Only:
-
-- `id` (String) Unique, immutable, system-controlled identifier of the firewall rule.
-- `time_created` (String) Timestamp of when this firewall rule was created.
-- `time_modified` (String) Timestamp of when this firewall rule was last modified.
```

Comment:
We should add these two to the root of the schema no?

Reply:
🙈 I totally thought I did that. Added!